tutorials/basic-topics/sorting/selection-sort.md

```diff
@@ -11,4 +11,110 @@ keywords:
   - selection sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Bobliuuu"/>
+
+## Overview
+
+Selection sort is a commonly used comparison-based sorting algorithm, that is also very simple to implement. 
+Selection sort works on the premise that two subarrays are maintained: one which is **sorted**, and one which is **unsorted**. 
+Each step, one more element of the array gets sorted, until the entire array is sorted. 
+
+- **subarray**: A subsequence or slice of an array
+
+## Algorithm
+
+The concept of selection sort is that each time, we find the minimum element in the unsorted subarray, and we put it at the end of the sorted subarray. 
+Consider an array with n distinct elements. Looping n times, we
+- Find the minimum element in the unsorted subarray
+- Move that element to the end of the sorted subarray
+
+The reason the algorithm works is that each time, the smallest value in the unsorted array is the largest value in the sorted array. 
+
+## Example: [75. Sort Colors](https://leetcode.com/problems/sort-colors/)
+
+An array of integers `nums` is given, with n values that are either 0, 1, or 2 (representing red, white, or blue). We have to sort them in place so that adjacent elements are together.
+In order to achieve this, we can use selection sort. 
+
+## Pseudocode
+
+We loop through the array once. 
```

Comment:
Pseudocode can't be paragraph, It should be detailed instructions line by line. 
Readers find difficulty reading this.